client/actions/main.ts

```diff
@@ -718,6 +718,11 @@ const openIgnoreCancelSaveModal = ({
             selectors.planning.storedPlannings(getState());
         const item = get(storedItems, itemId) || {};
 
+        if (itemType === ITEM_TYPE.EVENT) {
+            // Load associated plannings so they can be used later
+            dispatch(eventsApi.loadAssociatedPlannings(item));
```

Comment:
The modal would show before this request has finished, and then the modal will be re-drawn once this request finished.

This re-render can be avoided if we wait here for the Planning items to load first, then continue on with displaying the moal